.devcontainer/onCreateCommand.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd .install
+./install_script.sh
+./install_boost.sh 1.83.0
```

Comment:
can versions be moved somewhere else? we have these sprinkled in a few places

Reply:
I agree. If we can't use a GitHub environment argument here, maybe it is time we have a file for the boost version and read it from all the scripts that require it